app/views/groups/show.html.erb

```diff
@@ -1,4 +1,5 @@
 <div class="transaction">
+<h1 class='text-success mb-0'>$ <%= @group.total_amount %></h1>
```

Comment:
[REQUIRED] Good job adding the total amount but kindly ensure that it matches the template in terms of colour, typography and layout. Please put it in a separate rectangular section at the top, add some text on the left and the total amount on the right just like in the template image below:
![image](https://github.com/manzitresor/budget_app/assets/62938735/a98e7d2e-b83a-4927-9133-ede014dab349)